quiz.js

```diff
@@ -26,6 +38,16 @@ if (square(2) !== 4 || cube(3) !== 27 || quad(4) !== 256) {
 
 /* your code starts here */
 
+function sum(numbers, twiceOrHalf) {
+  if (twiceOrHalf === true) {
```

Comment:
this is fine, but twiceOrHalf is truthy by itself, so you dont need to compare it to true. next time try using `if (twiceOrHalf){ .. };` instead